Package.swift

```diff
@@ -120,7 +123,7 @@ for target in package.targets where target.isTest || target.name == "CXTestUtili
     target.swiftSettings.append(contentsOf: combineImp.swiftSettings)
 }
 
-if combineImp == .combine && env["CX_CONTINUOUS_INTEGRATION"] != nil {
+if combineImp == .combine {
```

Comment:
So client can't use CXShim on lower version OS 🤔?

Reply:
If he is using `combine`...

Reply:
`canImport(combine)` doesn't mean you can use combine, you still need to update the deployment target to macos10.15+/ios13+

Reply:
We do need "CX_CONTINUOUS_INTEGRATION"!